build.gradle

```diff
@@ -416,7 +416,7 @@ tasks.register('buildJniLib', Exec) {
     args.add("--build")
     args.add("jni/build")
     args.add("--target")
-    def knn_libs = ['opensearchknn_faiss', 'opensearchknn_common', 'opensearchknn_nmslib']
+    def knn_libs = ['opensearchknn_simd', 'opensearchknn_faiss', 'opensearchknn_common', 'opensearchknn_nmslib']
```

Comment:
Not directly related to this - but can you also add in support for a flag to disable SIMD.
It can be an index setting or an arg

```
-Dknn.simd.enabled=false
```

For scenarios where we want to avoid SIMD completely - an example can be compatibility issues with a version upgrade/missing libs. It'll be good to have a lever.